store/sqlstore/file_info_store.go

```diff
@@ -111,35 +99,10 @@ func (fs SqlFileInfoStore) GetByPath(path string) (*model.FileInfo, *model.AppEr
 	return info, nil
 }
 
-func (fs SqlFileInfoStore) InvalidateFileInfosForPostCache(postId string) {
-	fileInfoCache.Remove(postId)
-	if fs.metrics != nil {
-		fs.metrics.IncrementMemCacheInvalidationCounter("File Info Cache - Remove by PostId")
-	}
+func (fs SqlFileInfoStore) InvalidateFileInfosForPostCache(postId string, deleted bool) {
```

Comment:
Do we need to keep this here? It seems that now it should be calling the one in `LocalCacheFileInfoStore`

Reply:
Yes, because is part of the Store interface, so we need to implement it (in this case with an empty implementation).